realm/register/register.gno

```diff
@@ -0,0 +1,57 @@
+package register
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/demo/avl"
+)
+
+var players = avl.Tree{} // player -> token
+
+func RegisterPlayer(playerAddr std.Address, token string) {
+	assertIsFaucet()
+
+	if IsAddressRegistered(playerAddr) {
+		panic("faucet already used")
```

Comment:
This actually changes the current behavior of the gnochess faucet.

Currently, the faucet is invoked each time the user refreshes the website, and potentially, if his account balance is less than `fundLimit`, he's re-funded.

Here it seems like a user can't never be re-funded.

Reply:
Wrong statement, re-fund never happens currently because the token middleware quits abruptly with `Already claimed` error.